### DISCUSSION TITLE
how to use natural logarithm operator ('ln') in an expression of function?

### DISCUSSION THREAD

**Discussion by ananthanarasimhanj** [idaholab/moose#27416] [Q&A General]
what is the syntax to use how to use natural logarithm operator ('ln') in an expression of function? let say example.. ln(x).

**Comment by GiudGiud** [comment]
Hello

You mean an expression in a ParsedFunction?

You can check the fparser documentation, it s linked in the ParsedFunction moose documentation. 

I would just do log(x)/log(10) and move on

Guillaume
